src/views/clusteroverview/SettingsTab/UserTab/components/TaskPermissionsSection/TaskPermissionsSection.tsx

```diff
@@ -14,7 +14,7 @@ const TaskPermissionsSection: FC = () => {
   const { t } = useKubevirtTranslation();
   const { capabilitiesData, isLoading } = usePermissions();
   return (
-    <ExpandSection className="permissions-tab--main" toggleText={t('Permissions')}>
+    <ExpandSection toggleText={t('Permissions')}>
```

Comment:
please check if the class permissions-tab--main isnt needed anymore and if so delete it as well, no need for unused classes

Reply:
it's only used here, it looks fine to me by removing it just here.
```
src]# grep "permissions-tab--main" $(find . -type f)
grep: ./utils/components/HardwareDevices/list/hooks/useHardwareDevicesPageColumns: No such file or directory
grep: .tsx: No such file or directory
./views/clusteroverview/SettingsTab/UserTab/components/TaskPermissionsSection/TaskPermissionsSection.tsx:      <Divider className="permissions-tab--main__divider" />
./views/clusteroverview/SettingsTab/UserTab/components/TaskPermissionsSection/TaskPermissionsSection.tsx:          <Flex className="permissions-tab--main__row" key={taskName}>

```